docs/reference/filter_field_definition.rst

```diff
@@ -202,8 +202,8 @@ To make the filter case insensitive, use the ``compare_case_insensitive`` option
         protected function configureDatagridFilters(DatagridMapper $datagridMapper)
         {
             $datagridMapper
-                ->add('title', 'doctrine_phpcr_string', array('compare_case_insensitive' => true))
-                ->add('author', 'doctrine_phpcr_string', array('compare_case_insensitive' => true))
+                ->add('title', 'doctrine_phpcr_string', ['compare_case_insensitive' => true])
+                ->add('author', 'doctrine_phpcr_string', ['compare_case_insensitive' => true])
                 ->add('label', 'doctrine_phpcr_string')
```

Comment:
Statt `doctrine_phpcr_string` sollte ....::class genutzt werden inklusive dem use use Statement

Reply:
I changed them all, but this can be tricky again now.